lemur/utility.py

```diff
@@ -295,23 +311,24 @@ def serialize_experiment_list(experiment_list):
     return experiments
 
 
-def serialize_admin_list(admin_list):
-    current_admins = []
-    for admin in admin_list:
-        classes = [c.id for c in admin.classes]
-        labs = [lab.id for lab in admin.labs]
-        current_admins.append({'id': admin.id,
-                               'username': admin.username,
-                               'name': admin.name, 'classes': classes,
+def serialize_user_list(user_list):
+    user_json_list = []
+    for user in user_list:
+        classes = [c.id for c in user.classes]
+        labs = [lab.id for lab in user.labs]
+        user_json_list.append({'username': user.id,
```

Comment:
the mistake i'm trying to point out is that you call it json even though it is not. I agree that actually serializing it here would be a mistake.